src/lib/Persistence/Legacy/Content/URLWildcard/Query/CriterionHandler/LogicalAnd.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriterionHandler;
+
+use Doctrine\DBAL\Query\Expression\CompositeExpression;
+use Doctrine\DBAL\Query\QueryBuilder;
+use Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion;
+use Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriteriaConverter;
+use Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriterionHandler;
+
+final class LogicalAnd implements CriterionHandler
+{
+    public function accept(Criterion $criterion): bool
+    {
+        return $criterion instanceof Criterion\LogicalAnd;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
```

Comment:
`inheritdoc` is redundant and not needed, but we can help static analysis (looking at the PHPStorm's one rn), so it doesn't report a polymorphic call on `$criterion->criteria`

```suggestion
     * @param \Ibexa\Contracts\Core\Repository\Values\Content\URLWildcard\Query\Criterion\LogicalAnd $criterion

```